apps/wing/project-templates/typescript/empty/main.ts

```diff
@@ -0,0 +1,19 @@
+import { cloud, inflight, lift, main } from "@wingcloud/framework";
+
+main((root, test) => {
+  const fn = new cloud.Function(
+    root,
+    "Function",
+    inflight(async () => {
+      return "hello, world";
+    })
+  );
+
+  test(
+    "fn returns hello",
+    lift({ fn }).inflight(async ({ fn }) => {
+      const assert = await import("node:assert");
```

Comment:
Is this an optimization of sorts? I wonder if it be more natural to import node:assert at the top of the file

Reply:
It's a necessary evil. Nothing from the outer scope can be used, including imports.

I have an idea to do some special hoisting for symbols that come from import statements instead, but for now this is needed.